### PR TITLE
WT-4588 test/format should drop core even when Jenkins or connection terminates

### PR DESCRIPTION
**Pull request by keithbostic**

### CODE REVIEW

**Comment by keithbostic** [comment]
autobuild: retest this please

File: test/format/t.c [hunk 1]
```diff
@@ -38,6 +38,19 @@ static void usage(void)
 extern int __wt_optind;
 extern char *__wt_optarg;
 
+/*
+ * signal)handler --
```
Comment (agorrod) [inline test/format/t.c]: ) should be _

File: test/format/t.c [hunk 2]
```diff
@@ -51,6 +64,16 @@ main(int argc, char *argv[])
 
 	(void)testutil_set_progname(argv);
 
+#ifdef SIGALRM
```
Comment (agorrod) [inline test/format/t.c]: These ifdefs are surprising enough that I think they warrant a comment. Something like "Windows and Linux support a different set of signals, so be conservative about installing handlers."

File: test/format/t.c (hunk 1 above)
Comment (keithbostic) [reply test/format/t.c]: Thanks, fixed in e9f44ce1b.

File: test/format/t.c (hunk 2 above)
Comment (keithbostic) [reply test/format/t.c]: Added in e9f44ce1b.

**Pull request merged**